api/release_notes/2019-release-notes.md

```diff
@@ -31,10 +31,10 @@ rules/:rule_id?include=rule_components
 
 ### 3) Rule Components belong to Properties now
 
-Rule Components used exist as a child of a Rule.  That made it impossible to reuse the same components across multiple rules.  In order to enable us to add this capability in the future, Rule Components need to move out from under a Rule and live at the Property level.
+Rule Components used to exist as children of a Rule.  That made it impossible to reuse the same components across multiple rules.  In order to enable us to add this capability in the future, Rule Components need to move out from under a Rule and live at the Property level.
 
 Please see [Rule Components](/api/reference/1.0/rule_components/) for detailed examples.
 
 ### 4) Hosts
 
-Most everyone we talked to told us that the name Adapter didn't make much sense.  Much debate ensued.  Adapters are now called Hosts.  Please see [Hosts](/api/reference/1.0/hosts/) for detailed examples.
\ No newline at end of file
+Most everyone we talked to told us that the name Adapter didn't make much sense.  Much debate ensued.  Adapters are now called Hosts.  Please see [Hosts](/api/reference/1.0/hosts/) for detailed examples.
```

Comment:
`Most of the people we talked to`

Reply:
I'm not editing for tone, I'm just fixing errors. (BTW: that text only showed up in the diff because vi automatically "fixes" files whose last line has no terminating newline.)